docs/introduction/install.rst

```diff
@@ -2,67 +2,54 @@
 Install Auspice
 ===============
 
-Prerequisites
-=============
+.. note::
+   This is an Auspice-specific installation guide. If you wish to use Nextstrain as a whole, please refer to `the Nextstrain installation guide <https://docs.nextstrain.org/en/latest/install.html>`__.
```

Comment:
contextualize with link

```suggestion
   This is an Auspice-specific installation guide. If you wish to use Nextstrain :doc:`as a whole <docs.nextstrain.org:learn/parts>`, please refer to `the Nextstrain installation guide <https://docs.nextstrain.org/en/latest/install.html>`__.
```

Reply:
Done in [force-push](https://github.com/nextstrain/auspice/compare/5c0f03aa0a8ea792c10bfee4392aca7db87b5bc7..411324b085ac9b6e61aa02cdb082362073d45f95). Also updated the other link to use `:doc:`.